app-center-webapps/src/main/resources/locale/navigation/portal/administration_en.properties

```diff
@@ -0,0 +1,19 @@
+#
+# This file is part of the Meeds project (https://meeds.io/).
```

Comment:
- update header
- you should declare the file into https://github.com/Meeds-io/app-center/blob/feature/experience/translations.properties#L22 like done for global site, in order to be considered by crowdin